backend/pkg/api/types/data_access.go

```diff
@@ -331,3 +352,169 @@ const CtxUserIdKey CtxKey = "user_id"
 const CtxIsMockedKey CtxKey = "is_mocked"
 const CtxMockSeedKey CtxKey = "mock_seed"
 const CtxDashboardIdKey CtxKey = "dashboard_id"
+
+// -------------------------
+// Search structs
+
+// -- General
+type SearchType int
+
+// all possible search types
+const (
+	SearchTypeInteger SearchType = iota
+	SearchTypeName
+	SearchTypeEthereumAddress
+	SearchTypeWithdrawalCredential
+	SearchTypeEnsName
+	SearchTypeGraffiti
+	SearchTypeEmail
+	SearchTypePassword
+	SearchTypeEmailUserToken
+	SearchTypeJsonContentType
```

Comment:
Why are these valid search types?

Reply:
They are not, just copied all regexes for now